api/server/fns_create.go

```diff
@@ -26,5 +27,17 @@ func (s *Server) handleFnCreate(c *gin.Context) {
 		handleErrorResponse(c, err)
 	}
 
+	app, err := s.datastore.GetAppByID(ctx, fnCreated.AppID)
```

Comment:
Why are we fetching it again from the database?

Reply:
We do not have the app.. Just the appID and annotators take the full app.

Reply:
As to if we can make the annotators not require full app that is a different question that I have no clue as to the answer for.

Reply:
it's okay for this endpoint to look up the app after create, though it would be nice if we made sure that if the resource gets created we return 202/200 instead of error'ing for e.g. app lookup/trigger (for GET this is less of an issue since it's a read-only endpoint)

Reply:
I was wondering if we can avoid another lookup of app in the same request flow, perfectly fine if not.

Reply:
not really without changing the annotator contract, which maybe should be more flexible (punt), but right now it's not. this endpoint is pretty low throughput and not necessarily latency sensitive at that level though, so it's ok I think

Reply:
Ill fix the resp codes. @rdallman